ui/update_status.go

```diff
@@ -105,13 +99,12 @@ func (ui *UserInterface) updateStatus(sleepTime time.Duration) {
 
 		// Update the "volume" view:
 		if castVolume != nil {
+			ui.volume = int(castVolume.Level * 100)
+			ui.muted = castVolume.Muted
+
 			viewVolume.Clear()
 			if ui.muted {
-				if castVolume.Muted {
-					fmt.Fprintf(viewVolume, "%s(muted)", volumeMutedColour)
-				} else {
-					fmt.Fprintf(viewVolume, "%s(muted)", volumeColour)
-				}
+				fmt.Fprintf(viewVolume, "%s(muted)", volumeMutedColour)
```

Comment:
I don't fully understand why `(muted)` was printed in a different color depending on the status message, but regardless the else case was never going to be used anymore since ui.muted is assigned to the value of castVolume.muted earlier now.

Reply:
I am not too sure either. But this looks good to me!